src/main/resources/script_templates/Neuroanatomy/pyimagej/pyijtest.py

```diff
@@ -1,67 +1,82 @@
+# -*- coding: utf-8 -*-
+"""
+This python script demonstrates how to integrate functionality from
+Simple Neurite Tracer with various python libraries through pyimagej.
+It requires that the latest version of SNT is installed in your
+ImageJ/Fiji environment.
+
+Python dependencies:
+- pyimagej
+- numpy
+- scipy
+"""
+
 import imagej
 import numpy as np
-import ast
 from scipy.spatial import ConvexHull
 
-ij = imagej.init(r'C:\Users\cam\Desktop\Fiji.app', headless=False)

```

Comment:
It would be useful to mention that this can be ` ij = imagej.init('sc.fiji:fiji')` once the scijava branch gets merged into master

Reply:
done